adoption-metrics/define_adoption_metrics.md

```diff
@@ -0,0 +1,56 @@
+# What adoption metrics Packit team wants to measure
+
+## Static meaurement (updated periodically)
+
+### Newcomers adoption
+
+- Total amount of newcomers in a given period (monthly/quarterly/half yearly/yearly).
+- How many newcomers (percentage) from a given period already tried some functionality that
+  Packit(-service) provides (COPR, TF, all downstream jobs).
+- How fast newcomers are? How much time (in average) they need from approval to actually
+  do something?
+- How often are different services (COPR, TF, all downstream jobs) used by newcomers?
+  What they use most often and what they don't use at all?
+
+### Active users
+
+Active users are those from whom we have recorded at least one activity in the past period
+and they do not fit into the category of newcomers.
+
+- How often are different services (COPR, TF, all downstream jobs) used by active users?
+  What they use most often and what they use less?
+- How frequently active users use packit services?
+
+### Users retention
+
+- Percentage of previous newcomers who became active Packit users.
+  ((amount of newcomers from previous period becoming an active users
+  / amount of newcomers from previous period) \* 100)
```

Comment:
We should somehow get rid of the projects that have the Packit installed only because of the `install GitHub application to all repositories`. We can check the presence of Packit config or the namespace to be approved. It's also important who we consider an active user.

Reply:
I'll add a definition of newcomer - "Newcomers are users who have an approved namespace". 
Would it be enough?

Reply:
And are we going to count users or approved namespaces?

Reply:
Hmmm that's true, (for me) it makes sense to count approved namespace (I made the equation user == namespace in my head. Sorry for the confusion)